workspaces/announcements/plugins/announcements-backend/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@backstage-community/plugin-announcements-backend",
+  "version": "0.9.3",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "MIT",
```

Comment:
@vinzscam is there any issues with this being MIT? Backstage itself and best I know the plugins here are Apache 2.0

Reply:
this can be updated to reflect the standards of `backstage/community-plugins`